x-pack/plugins/fleet/common/services/decode_cloud_id.ts

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+const CLOUD_DEFAULT_PORT = '9243';
```

Comment:
out of scope for this PR, but this kind of information seems like it should be exported from `cloud` plugin

Reply:
Yes actually the whole `decode_cloud_id` service should probably exported by the cloud plugin.